py-scoring/conda/h2o_pysparkling_scoring_SUBST_SPARK_MAJOR_VERSION/meta.yaml

```diff
@@ -16,5 +16,5 @@ requirements:
 
 about:
   home: https://github.com/h2oai/sparkling-water.git
-  license: Apache License Version 2.0
-  license_family: Apache
+  license: AGPL-3
```

Comment:
Shouldn't this be `GNU Affero General Public License v3`? https://github.com/h2oai/h2o-3/pull/15533/files#diff-88f12fed5fbaae7ae5bb9d65df632b469aad6f69e7bcadcbfda31939b01c47c3R25